cogs/voice.py

```diff
@@ -0,0 +1,113 @@
+from discord.ext import commands
+from lib.voice_server import VoiceServer
+import discord
+import asyncio
+import sentry_sdk
+
+
+class Voice(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def join(self, ctx):
+        async with ctx.channel.typing():
+            try:
+
+                # サーバーがすでに接続されていた場合
+                if self.bot.voice_manager.get(ctx.guild.id) is not None:
+                    await ctx.send("このサーバーはすでに接続されています。moveコマンドを使用するか、切断してください。")
+                    return
```

Comment:
「サーバーが接続されている」はおかしいので、こうじゃないですかね

```suggestion
                # サーバー内で既に利用されていた場合
                if self.bot.voice_manager.get(ctx.guild.id) is not None:
                    await ctx.send("このサーバー内で既に利用されています。moveコマンドを使用するか、切断してから再度お試しください。")
                    return
```

Reply:
そうですね、ありがとうございます

Reply:
修正しました🙏